app/pb_kits/playbook/pb_date/_date.jsx

```diff
@@ -1,21 +1,59 @@
+/* @flow */
+
 import React from 'react';
-import PropTypes from "prop-types";
-
-const propTypes = {
-  className: PropTypes.string,
-  id: PropTypes.string
-};
-
-class Date extends React.Component {
-  render() {
-    return (
-      <div className="pb_date">
-        <span>DATE CONTENT</span>
-      </div>
-    )
-  }
+import DateTime from '../pb_kit/dateTime.js'
+import moment from 'moment'
+require('moment-strftime')
+require('moment-timezone')
+
+import {
+  Body,
+  Icon,
+} from '../'
+
+type DateProps = {
+  className: String,
```

Comment:
It seems like the `className` prop is never used in the component's implementation.